src/components/Security/Profiles/Update.vue

```diff
@@ -14,13 +14,21 @@
   </b-container>
 </template>
 
+<style lang="scss" scoped>
```

Comment:
IMHO we can use Bootstrap classes: `d-flex flex-column h-100`
Let's keep the custom classes for when we really can't do without